plugins/source/aws/docs/tables/aws_resiliencehub_apps.md

```diff
@@ -0,0 +1,35 @@
+# Table: aws_resiliencehub_apps
+
+https://docs.aws.amazon.com/resilience-hub/latest/APIReference/API_App.html
+
+The composite primary key for this table is (**account_id**, **region**, **arn**).
```

Comment:
Assuming the `arn` contains account ID and region components, we should be able to drop account ID and region from the composite PK and only keep `arn`.